### ISSUE TITLE
Can't install through Bower

### ISSUE THREAD

**Issue by paquitodev** [formvalidation/formvalidation#1334]
@nghuuphuoc I have tried this lines on my `bower.json` file:

    "formvalidation": "~0.6.0",
    "bootstrapvalidator": "~0.6.0",

And none install the latest 0.6.0, any advice?

**Comment by vincentjames501** [comment]
+1. Same with "form.validation"

**Comment by paquitodev** [comment]
@vincentjames501 I have managed to install through `master` but this is, perhaps, not the best way since master should be the development branch and not the stable or perhaps I'm complete wrong at this point, anyway lets see what developer has to said around this

**Comment by shamiao** [comment]
+1, 0.6.0 is also not released on github. 
@nghuuphuoc

**Comment by snrbrnjna** [comment]
you need to tag your releases properly for bower to distribute the latest version.

you can achieve this easily with a grunt plugin: https://github.com/vojtajina/grunt-bump

I'll send you a pull request.

**Comment by rosskevin** [comment]
Agreed, added `"formValidation": "formvalidation/formvalidation#master"` in my bower.json for now, but I don't want to inadvertently pull an unstable source.

**Comment by rosskevin** [comment]
I had to abandon `master` as there were some bugs disabling the submit button.  This one appears to be the 0.6.0 commit: 			

```
"formValidation": "formvalidation/formvalidation#77d4e88f3fcdb70e3f0a5e65e40a560ca1de47c4",
```

**Comment by nghuuphuoc** [comment]
@rosskevin 
The submit button enabled/disabled behavior has been fixing in v0.6.1-dev.
Since you're a customer, just let me know if you want to use v0.6.0. I will use the download system to give you a link via email.

**Comment by paquitodev** [comment]
So, I should change to `0.6.1-dev` to fix the issues? Is that a tag at 
Bower?

**Comment by vincentjames501** [comment]
+1. I don't see a tag or branch for 0.6.1-dev and the submit button error is also affecting us. I'm also a customer @nghuuphuoc, can you please advise how to get bower to install a working version?
